avr/cores/megacommand/MCL/MCLMemory.h

```diff
@@ -12,15 +12,36 @@
 
 //#define MD_TRACK_LEN (sizeof(GridTrack) + sizeof(MDSeqTrackData) + sizeof(MDMachine))
 //#define A4_TRACK_LEN (sizeof(GridTrack) + sizeof(ExtSeqTrackData) + sizeof(A4Sound))
+
+//Use these to produce compiler errors that probes the sizes!
+//template<int X> struct __WOW;
+//__WOW<sizeof(MDTrackLight)> szmd;
+//__WOW<sizeof(A4Track)> sza4;
+#pragma message("MD_TRACK_LEN = 501")
```

Comment:
Should this not be hardcoded, or have I misunderstood the intention here?

Reply:
during compile-time, sizeof will not calculate the actual value in a macro. To get it at compile time, uncomment the templated probe `__WOW`, but it produces compiler errors to show the info :O

so if we want to update this value, we have to stop the pipeline, produce some errors, extract the value and hardcode it back here. anyway it's compile time stuff.